src/lib/containers/HasAccess.tsx

```diff
@@ -31,7 +31,6 @@ export type HasAccessProps = {
   fileHandle?: FileHandle
```

Comment:
rendering is different between has access and synapse table, dependent on whether the filehandle is present or not.